libs/ActivityKit/source/ActivityKit.cpp

```diff
@@ -0,0 +1,31 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "ActivityKit.h"
+
+using namespace leka;
+
+void ActivityKit::registerActivities(std::unordered_map<MagicCard, interface::Activity> const &activities)
+{
+	_activities = activities;
+}
+
+void ActivityKit::start(const MagicCard &card)
+{
+	stop();
+
+	_activity = &_activities.at(card);
```

Comment:
pas besoin du `&` avec le pointeur

```suggestion
	_current_activity = _activities.at(card);
```